### PR TITLE
initial implementation of  AGR-1160 - google structured data

### PR DESCRIPTION
**Pull request by nathandunn**
just a reference for this proposal:

https://agr-jira.atlassian.net/browse/AGR-1160

will close for now, but I find its easier to look at code.

Would need to replicate for the disease page if we go forward.

### CODE REVIEW

**Comment by nathandunn** [comment]
Just creating for a reference.

**Comment by nathandunn** [comment]
To make this work, I would:
-  change the @link to “url” 
- [ ] make description match name or gene description (at least until we get a gene description in there)
- [ ] make keywords contain the species, gene name, and gene
- [ ] org name is some version of “The Alliance of Genome Resources"
- [ ] version (2.0)
- [ ] license (whatever we are using): https://creativecommons.org/licenses/by/4.0/ CC BY 4.0

The final product would include what Sarala has below:

```
  "@context": "http://schema.org",
  "@type": "Dataset",
  "name": "The name of the item",
  "description": "A description of the item",
  "url": "URL of the item",
  "keywords": "Keywords or tags used to describe this item",
  "includedInDataCatalog": "URL of the containing DataCatalog",
  "creator": {
    "@type": "Organization",
    "name": "Organization name"
  },
  "version": "Version Number",
  "license": "License names"
```

**Comment by christabone** [comment]
Have we agreed upon a license for the Alliance?

**Comment by nathandunn** [comment]
Stacia say it was the one I put in but we can clarify. 

Nathan

**Comment by srengel** [comment]
this license: https://creativecommons.org/licenses/by/4.0/ CC BY 4.0

File: src/components/headMetaTags.js [hunk 1]
```diff
@@ -16,14 +41,17 @@ class HeadMetaTags extends Component {
           ]}
           title={title}
         />
+        {schema &&
+        <Helmet script={[schema]} />
+        }
```
Comment (paaatrick) [inline src/components/headMetaTags.js]: Do we need a separate `<Helmet>` component for this? Can't it just be included in the other one? React won't pass a prop to a component if the value is undefined.

**Comment by nathandunn** [comment]
Just a quick FYI.  We'll be talking about this on Tuesday (meeting in calender, but ping me for times) to get feedback on the implementation.   I expect we'll make further changes, though, sometime next week and we'll have the final implementation for 2.0.  

@a8wright  I adopted this approach based on feedback from folks at identifiers.org and the google structured data page.  Open to different implementations, though. 

@paaatrick  Good catch. I added the fix you recommended.  It does seem to blow up if I don't handle the undefined case, but maybe you'd intended something else.  With regard to the allele case, I think we should definitely handle that case when we get to it, but point taken.   We should have another update and then we can merge it in with your suggested hierarchy, which sounds more scalable.

**Pull request merged**